optaplanner-docs/src/modules/ROOT/pages/use-cases-and-examples/vaccination-scheduling/vaccination-scheduling.adoc

```diff
@@ -0,0 +1,266 @@
+[id="chap-optaplanner-vaccination_{context}"]
```

Comment:
I think the `context` attribute needs to be define at a higher level - in the file that includes this one `vaccination-scheduling.adoc`. Otherwise at this point it is undefined and the substitution here doesn't work.